test/unit/event.js

```diff
@@ -2821,6 +2821,46 @@ QUnit.test( "originalEvent property for Chrome, Safari, Fx & Edge of simulated e
 	jQuery( "#donor-input" ).trigger( "focus" );
 } );
 
+QUnit.test(
+	"navite stop(Immediate)Propagation methods should not be called from simulated event",
+	function( assert ) {
+		var userAgent = window.navigator.userAgent;
+
+		if ( !( /firefox/i.test( userAgent ) || /safari/i.test( userAgent ) ) ) {
+			assert.expect( 1 );
+			assert.ok( true, "Assertions should run only in Chrome, Safari, Fx & Edge" );
```

Comment:
Why is that necessary?

Also, it might be simpler to say where we don't want to run them. :) It seems only in IE?